docs/finality-provider-operation.md

```diff
@@ -731,7 +731,136 @@ The response includes:
   transaction, which you can use to verify the success of the transaction
   on the Babylon chain.
 
-### 6.2. Statuses of Finality Provider
+### 6.2 Rewards and Refunding
+
+Rewards are accumulated in a reward gauge, and a finality provider becomes
+eligible for a rewards gauge when it meets certain conditions, such as
+having an `ACTIVE` status, participating in voting, and obtaining
+delegations. The distribution of rewards is based on the provider's
+voting power portion relative to other voters.
+
+#### 6.2.1 Querying Rewards
+
+To query rewards of a given stakeholder address, use the following command.
+
+```shell
+fpd reward-gauges <address>
+```
+
+Parameters:
+
+* `<address>`: The Babylon address of the stakeholder in bech32 string.
+
+#### 6.2.2 Withdraw Rewards
+
+This command will withdraw all accumulated rewards. The `fpd` must be **stopped**
+before performing this action as it uses the registered key as the operational
+key. If the operator follows the steps in
+[6.2.4 Refunding finality provider](#624-refunding-finality-provider)
```

Comment:
Maybe we can put `fpd must be stopped...` as an important note after we show how the command is executed where we can say `Otherwise, account sequence mismatch error might be encountered because the key used for sending the withdrawal transaction is under use by the finality provider sending operational transaction. This issue will be resolved after following the setup instructions in 6.2.4....`